src/api/generate-xlsx/map-application-to-XLSX/map-policy/map-single-contract-policy/index.test.ts

```diff
@@ -51,7 +51,7 @@ describe('api/generate-xlsx/map-application-to-xlsx/map-policy/map-single-contra
     const result = mapSingleContractPolicy(policy);
 
     const expected = [
-      xlsxRow(String(FIELDS[CONTRACT_COMPLETION_DATE]), formatDate(policy[CONTRACT_COMPLETION_DATE], 'dd-MMM-yy')),
+      xlsxRow(String(FIELDS[CONTRACT_COMPLETION_DATE]), formatDate(policy[CONTRACT_COMPLETION_DATE], 'dd MM yyyy')),
```

Comment:
constant for this?

Reply:
Agree, can this be done after this PR though as there are 10 instances that need updating, don't want to make this PR even bigger

Reply:
Created a ticket for this, for next sprint, EMS-3482